typescript/types.md

```diff
@@ -178,7 +178,7 @@ following triple-slash directives near the top of the entry point file for the
 worker script, Deno will now type check it as a Deno worker script, irrespective
 of how the module is analyzed:
 
-```ts
+```ts ignore
 /// <reference no-default-lib="true" />
 /// <reference lib="deno.worker" />
 ```
```

Comment:
@bartlomieju I did some debugging and found out that these triple-slash directives are messing up the typechecking during `deno test --doc --unstable`! After ignoring it all the errors related to unstable APIs not being found even though we're passing --unstable are gone. :)

Reply:
Still 33 errors left, but they are 'legit'. I will try and fix some of them in the coming days